adcctestdata/dump_pyscf.py

```diff
@@ -217,5 +217,16 @@ def dump_pyscf(scfres, out):
     mmp.create_dataset("elec_1",
                        data=scfres.mol.intor_symmetric("int1e_r", comp=3))
 
+    varop = data.create_group("uncategorized_operators")
```

Comment:
British spelling, please: "uncategorised_operators"

Also: Better introduce groups "magnetic_moments" and "derivatives"? The problem is that changing the name of the group is a lot of hassle (it occurs in the DataHfProvider in adcc, the test setup, ...), , so we should choose the name wisely and "uncategorised_operators" opens the door to a lot of quick and dirty dumps of operators in the future, which I'm not sure is the wisest way to go.

Reply:
Oh come on British spelling...

Reply:
That's what's used basically everywhere in adcc. We can discuss about changing, but then you do the renaming ;).

Reply:
Done.

Reply:
I was just kidding... 😉